netaddr.go

```diff
@@ -846,3 +871,422 @@ func (p *IPPrefix) UnmarshalText(text []byte) error {
 func (p IPPrefix) String() string {
 	return fmt.Sprintf("%s/%d", p.IP, p.Bits)
 }
+
+// LastIP returns the last IP in the prefix.
+func (p IPPrefix) LastIP() IP {
+	if p.IP.IsZero() {
+		return IP{}
+	}
+	a16 := p.IP.As16()
```

Comment:
I have some bit twiddling math in wgengine/filter that would work a treat on these functions. Will send a followup PR to port things over.